badge/src/main/java/com/decathlon/vitamin/compose/badge/models/VitaminBadgeStyle.kt

```diff
@@ -0,0 +1,128 @@
+package com.decathlon.vitamin.compose.badge.models
+
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontWeight
+import com.decathlon.vitamin.compose.foundation.VitaminTheme
+
+/**
+ * Badge style
+ *
+ * @property backgroundColor
+ * @property textStyle
+ */
+data class BadgeStyle(
+    val backgroundColor: Color,
+    val textStyle: TextStyle,
```

Comment:
Move this on the size object (like other components) and rename `BadgeStyle` to `BadgeColors`

Reply:
I used badgeSize, badgeColor and textStyle (which need badge size to update textSize) instead. But if you think is not a good idea, perhaps we can rework it together